index.html

```diff
@@ -1,290 +1,326 @@
+        <img id="frameworks-image-title" src="./images/square-frameworks.png" alt="image of square" />Frameworks
       </div>
-      <hr />
-      <div class="technologies-about-me">
-        <div class="language-title">
-          <img
-            id="language-image-title"
-            src="./images/rectangle-languages.png"
-            alt="image of rectangle"
-          />Language
-        </div>
-        <div class="languages-container">
-          <ul class="languages-list">
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>Css</li>
-          </ul>
-        </div>
-        <div class="frameworks-title">
-          <img
-            id="frameworks-image-title"
-            src="./images/square-frameworks.png"
-            alt="image of square"
-          />Frameworks
-        </div>
-        <div class="frameworks-container">
-          <ul class="frameworks-list">
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </div>
-        <div class="skills-title">
-          <img id="skills-image-title" src="./images/circle-skills.png" alt="image of circle" />
-          Skills
-        </div>
-        <div class="skills-container">
-          <ul class="skills-list">
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
+      <div class="frameworks-container">
+        <ul class="frameworks-list">
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+      <div class="skills-title">
+        <img id="skills-image-title" src="./images/circle-skills.png" alt="image of circle" />
+        Skills
+      </div>
+      <div class="skills-container">
+        <ul class="skills-list">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+  <footer>
+    <section class="contact">
+      <p class="description-contact">
+        I'm always interested in hearing about
+        new projects, so if you'd like to chat please
+        get in touch.
+      </p>
+      <div>
+        <form action="https://formspree.io/f/mlezobkj" method="POST">
+          <ul class="contact-inputs">
+            <li>
+              <input type="text" name="full_name" id="full-name" maxlength="30" placeholder="Full Name" required>
+            </li>
+            <li>
+              <input type="email" name="email_address" id="email-address" placeholder="Email address" required>
+            </li>
+            <li>
+              <textarea name="message" maxlength="500" id="message" required>Enter your text here</textarea>
+            </li>
```

Comment:
- [ ] It would be better if you add a placeholder text here. :+1: 

Ex: `<textarea name="message" maxlength="500" id="message" placeholder="Enter your text here" required></textarea>`

Reply:
Thank for the comment, I will change that, you're right that looks better!